Source/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 
+include_directories_exported(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
It may be time to clean this up: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/CMakeLists.txt#L27 which is effectively adding same folder (though not exported to parent projects)